src/js/components/Calendar.js

```diff
@@ -46,12 +46,10 @@ var Calendar = React.createClass({
 
   componentDidUpdate: function (prevProps, prevState) {
     // Set up keyboard listeners appropriate to the current state.
-    if (! this.state.dropActive && prevState.dropActive) {
-      this._activation(this.state.dropActive);
-    }
-    if (this.state.dropActive && ! prevState.dropActive) {
-      this._activation(this.state.dropActive);
+    if (prevState.dropActive !== this.state.dropActive) {
+      this._activation(prevState.dropActive || this.state.dropActive);
```

Comment:
Line 49 looks good. However, I think this line is supposed to always pass this.state.dropActive, as this._activation handles both activation and deactivation and needs the undefined value to remove the added keyboard listeners.